src/main/java/com/minecolonies/coremod/client/gui/WindowCitizen.java

```diff
@@ -313,19 +313,19 @@
     /**
      * Scrollinglist of the resources.
      */
-    private final ScrollingList resourceList;
+    private final ScrollingList   resourceList;
     /**
      * Inventory of the player.
      */
-    private final InventoryPlayer inventory = this.mc.player.inventory;
+    private final InventoryPlayer inventory  = this.mc.player.inventory;
     /**
      * Is the player in creative or not.
      */
-    private final boolean isCreative = this.mc.player.capabilities.isCreativeMode;
+    private final boolean         isCreative = this.mc.player.capabilities.isCreativeMode;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1170)